### DISCUSSION TITLE
149.6版本再次出现ollama连接问题[Bug]

### DISCUSSION THREAD

**Discussion by BBM-Freehub** [lobehub/lobehub#2224] [General Question] [answered]
### 💻 系统环境

Ubuntu

### 📦 部署环境

Other

### 🌐 浏览器

Chrome

### 🐛 问题描述

相同的配置，在149.6版本再次出现ollama无法连接的问题。
<img width="935" alt="146" src="https://github.com/lobehub/lobe-chat/assets/86555393/6e2998a9-14e1-4740-9c8d-a1f7ccf38586">
注：0.149.5版本依旧可以正常连接ollama。

[a comment by an automated account is not shown]

**Comment by seasoncool** [comment]
关注一下，也碰到同样问题。
版本：LobeChat v0.149.6

[a comment by an automated account is not shown]

**Comment by liuzj288** [comment]
> ### 💻 系统环境
> Ubuntu
> 
> ### 📦 部署环境
> Other
> 
> ### 🌐 浏览器
> Chrome
> 
> ### 🐛 问题描述
> 相同的配置，在149.6版本再次出现ollama无法连接的问题。 <img alt="146" width="935" src="https://private-user-images.githubusercontent.com/86555393/325992819-6e2998a9-14e1-4740-9c8d-a1f7ccf38586.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxNDE3MjMsIm5iZiI6MTcxNDE0MTQyMywicGF0aCI6Ii84NjU1NTM5My8zMjU5OTI4MTktNmUyOTk4YTktMTRlMS00NzQwLTljOGQtYTFmN2NjZjM4NTg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI2VDE0MjM0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxNzYxNmViOThlZjI5YjMyNDNmYzU3YThjNzk1MDA1NWExOTYxYTMwYjMwZTliMDcyMDA3MTA3OGE4Yjg5YmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qqIfVjWbsRJd3f2f1yxK029mXPLPv8hSZZYyqHzbk1c"> 注：0.149.5版本依旧可以正常连接ollama。
> 
> ### 🚦 期望结果
> _No response_
> 
> ### 📷 复现步骤
> _No response_
> 
> ### 📝 补充信息
> _No response_

你的访问地址用的http还是https

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
去掉 `v1` 试试

[a comment by an automated account is not shown]

**Comment by BBM-Freehub** [comment]
> 去掉 `v1` 试试

非常感谢，去掉v1后，通过检测。

[a comment by an automated account is not shown]
